src/actions/content/content.js

```diff
@@ -150,7 +150,9 @@ export function getContent(
           b_size: settings.defaultPageSize,
         }
       : {},
-    settings.isMultilingual ? { expand: 'translations' } : {},
+    settings.isMultilingual
+      ? { expand: 'translations,navroot' }
+      : { expand: 'navroot' },
```

Comment:
Could it be added via the apiExpanders config setting, instead of here? That would make it easier to turn off for a large site that is trying to minimize the number of backend requests needed to render a page.

Reply:
Yes, this is something I wanted to discuss, specifically regarding performance.

We have code spread through Volto doing the ```if !hasApiExpanders(...)``` and first I thought to write this like that. 

On the other hand I saw that this ```navroot``` call was being done in every request after implementing this PR, so I thought it was worth to request the navroot expander by default. Moreover, I initially thought that we should be doing the same for at least the breadcrumbs expander too.

Then I saw [this config.settings setting called ```apiExpanders```](https://github.com/plone/volto/blob/master/src/config/index.js#L78) which really does what I say (expand the query by default with given expanders), but it is not used at all in plain Volto, which makes me think...

I would say (but that's perhaps a biased opinion because of this PR), that Volto should by default use that apiExpanders settings, to expand by default breadcrumbs, navroot, site and navigation, which are used anyway to build a stock Volto site, but let disable them using the config. This way the code could be written defensively using the ```is !hasApiExpanders```...

I don't know... I am open to discuss.

Reply:
Yes, I think we want to use apiExpanders by default for the expanders that are needed everywhere, but we were just proceeding cautiously. Right @sneridagh? Can we add to the default expanders for volto 17 or is there a reason to still be cautious?

Reply:
I have reworked this. I have gone the `!hasApiExpanders` way and let as it is. We can come to the use of `apiExpanders` setting later :)

Reply:
I'd introduce the expanders "all in" right now. There's nothing that prevents us for doing it in 17.